docker/mlpa/Dockerfile

```diff
@@ -54,9 +54,17 @@ COPY --from=build-env /go/bin/mlpab mlpab
 COPY --link web/template web/template
 COPY --link lang lang
 
-RUN addgroup -S app && \
-  adduser -S -g app app && \
-  chown -R app:app mlpab web/template web/static web/robots.txt
-USER app
+
+RUN addgroup -S user && \
+  adduser -S -g user user && \
+  chown -R user:user mlpab web/template web/static web/robots.txt
+
+COPY scripts/docker_hardening/alpine_image_hardening.sh /harden.sh
+
+RUN /harden.sh && rm /harden.sh
+
+USER user
```

Comment:
Might want to make this user more specifically named, like `mlpaapp` or similar?

Reply:
Ah, I see. Maybe `appuser` then. Naming to `user` just feels a bit unclear on intent to me.

Reply:
I need the name of the user to be agnostic of the container it's used in. The intent of the user is seen a couple of lines above where its given ownership of the the container's app code